lib/chat_api/slack.ex

```diff
@@ -180,46 +196,92 @@ defmodule ChatApi.Slack do
       end
 
     url = base <> "/conversations/" <> conversation_id
-    conversation = "<#{url}|conversation>"
+    dashboard = "<#{url}|dashboard>"
 
-    subject =
-      "New #{conversation} started with #{identify_customer(customer)}!" <>
-        "\n\nReply to this thread to start chatting :rocket:"
-
-    subject
+    "*:wave: #{identify_customer(customer)} says*: #{text}" <>
+      "\n\nReply to this thread to start chatting, or view in the #{dashboard} :rocket:"
   end
 
-  # TODO: show sender/customer name here if available rather than generic "Agent"/"Customer"
-  def get_slack_message_subject!(type, customer, _conversation_id, _thread) do
+  def get_message_text(%{
+        customer: customer,
+        text: text,
+        type: type,
+        conversation_id: _conversation_id,
+        thread: _thread
+      }) do
     case type do
-      :agent -> ":female-technologist: Agent:"
-      :customer -> ":wave: #{identify_customer(customer)}:"
+      :agent -> "*:female-technologist: Agent*: #{text}"
+      :customer -> "*:wave: #{identify_customer(customer)}*: #{text}"
       _ -> raise "Unrecognized sender type: " <> type
     end
   end
 
-  def get_slack_message_payload(subject, channel, text, nil) do
+  def get_message_payload(text, %{
+        channel: channel,
+        customer: %{
+          name: name,
+          email: email,
+          current_url: current_url,
+          browser: browser,
+          os: os
+        },
+        thread: nil
+      }) do
     %{
       "channel" => channel,
-      "text" => subject,
-      "attachments" => [%{"text" => text}]
+      "blocks" => [
```

Comment:
this looks like:
<img width="686" alt="Screen Shot 2020-08-25 at 10 36 16 AM" src="https://user-images.githubusercontent.com/5264279/91188044-d92cfb80-e6be-11ea-9dab-7a6763c199c5.png">